### DISCUSSION TITLE
Lets Have a Discord Community Server

### DISCUSSION THREAD

**Discussion by ryuusama09** [Kong/insomnia#5362] [Ideas]
Just a small proposal! :
Can we have a discord server for insomnia community.
Reasons to have :
- Better interaction and coordination 
- easy for newcomers like me to ask doubts and grow
- also have fun while

**Comment by filfreire** [comment]
Hi @ryuusama09 

We don't have any official support for Discord at the moment, in the meantime you can use Insomnia's Slack: https://chat.insomnia.rest/
